apps/server/src/modules/collaborative-text-editor/api/collaborative-text-editor.controller.ts

```diff
@@ -0,0 +1,42 @@
+import { Authenticate, CurrentUser, ICurrentUser } from '@modules/authentication';
+import { Controller, ForbiddenException, Get, NotFoundException, Param, Res } from '@nestjs/common';
+import { ApiOperation, ApiResponse, ApiTags } from '@nestjs/swagger';
+import { ApiValidationError } from '@shared/common';
+import { Response } from 'express';
+import { CollaborativeTextEditorUc } from './collaborative-text-editor.uc';
+import { CollaborativeTextEditorResponse } from './dto/collaborative-text-editor.response';
+import { GetCollaborativeTextEditorForParentParams } from './dto/get-collaborative-text-editor-for-parent.params';
+import { CollaborativeTextEditorMapper } from './mapper/collaborative-text-editor.mapper';
+
+@ApiTags('CollaborativeTextEditor')
+@Authenticate('jwt')
+@Controller('collaborative-text-editor')
+export class CollaborativeTextEditorController {
+	constructor(private readonly collaborativeTextEditorUc: CollaborativeTextEditorUc) {}
+
+	@ApiOperation({ summary: 'Redirect to CollaborativeTextEditor' })
```

Comment:
This sounds like the action would actually respond with a redirect, but that's not true. Is this summary really needed anyway?